src/frontend/components/shared/Modal.tsx

```diff
@@ -16,11 +16,12 @@ import { ESCAPE_KEY_CODE } from '../../constants/misc'
 
 interface IModalContent {
   show?: boolean
+  isHome?: boolean
```

Comment:
Gotcha yeah this makes more sense then. Can you change this from `isHome` to `isThin` so that it could be used more generally? Also 10% might be a little thin—can you include a screenshot in the PR?